MAINTAINERS.md

```diff
@@ -0,0 +1,40 @@
+## Maintainers
+Documentation for maintainers.
+
+Current maintainers: 
+- [Jake Bolam](https://github.com/jakebolam)
+- [Maximilian Berkmann](https://github.com/Berkmann18) 
+- [Tyler Benning](https://github.com/tbenning) 
+- [Jeff Wen](https://github.com/sinchang)
+
+See [the github All Contributors Core Team group](https://github.com/orgs/all-contributors/teams/core/members)
+
+## Roadmap
+- All Contributors Bot in Marketplace (Requires 250 bot installs)
+- All Contributors Website Portal?
+
+## Metrics OKR Period - Open Leaders (Mid Feb 2019 until April 2019)
+### Project & Website
+Date | Maintainers | Contributors |Stars | Website Languages  | Website Sessions | Seo Hits
+---|---|---|---|---|---|---
+**TARGET for April 201** | 4 | 40 | 2700 | 10 | 500 | 10
+Wed Feb 20th | 4 | 34 | 2442 | 7 | 382 | 0
+Wed Feb 13th | 4 | 33 | 2430 | 6 | 431 | 0
+
+
+### Bot
+Date | Bot Installs | Bot Stars | Bot Messages | Bot PRs 
+---|---|---|---|---
+**TARGET for April 2019** | 250 | 100 | 50 | 40
```

Comment:
Wanting to be able to measure our great progress!

Thinking of recording metrics, and setting targets? Thoughts?

Reply:
I'm cool with that, I suppose Sentry offers a way to get those metrics onto a file or something which could then be used for this document.